frontend/src/metabase/nav/components/SearchBar.jsx

```diff
@@ -128,6 +128,7 @@ export default class SearchBar extends React.Component {
             pl={1}
             ref={ref => (this.searchInput = ref)}
             value={searchText}
+            maxLength={200}
```

Comment:
@kdoh does this seem reasonable? I'm not sure there's a magic number

Reply:
Seems fair I think.